.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ env:
   # the Docker daemon only downloads 3 layers concurrently which prevents the other pull from making any progress.
   # This value should be greater than the time taken for the longest image pull.
   TESTCONTAINERS_PULL_PAUSE_TIMEOUT: 600
-  TESTCONTAINERS_SKIP_ARCHITECTURE_CHECK: true
+  TESTCONTAINERS_SKIP_ARCHITECTURE_CHECK: false
```

Comment:
We skip this in the CI because it requires to download all test images in the build-pt job which is expensive and fills up the runners disk.

Reply:
Please add a comemnt

Reply:
It's used in the `Build PT matrix (impacted-features)` step. The Python script calls `ptl describe`, which instantiates all test environments, including testcontainers image objects. 

`TESTCONTAINERS_NEVER_PULL` is independent of `TESTCONTAINERS_SKIP_ARCHITECTURE_CHECK`. `TESTCONTAINERS_NEVER_PULL` is used in a testcontainers pull policy, where `TESTCONTAINERS_SKIP_ARCHITECTURE_CHECK` is used by an image filter implemented by us, and it doesn't use testcontainers pull policy at all.